src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -299,17 +299,20 @@
 <div class="rating-container">
     <h2>Rating</h2>
     <h5>Large stars</h5>
-    <BitRating DefaultRating="5" Size="RatingSize.Large" />
+    <BitRating Size="RatingSize.Large" @bind-Value="RatingDefaultValue" />
     <h5>Small stars, with 0 stars allowed</h5>
-    <BitRating AllowZeroStars="true" />
+    <BitRating AllowZeroStars="true" @bind-Value="RatingDefaultValue" />
     <h5>10 small stars</h5>
-    <BitRating Max="10" />
+    <div>
+        <BitRating Max="10" OnChange="@(arg => OnRatingChange(arg))" @bind-Value="RatingBoundValue" />
+        <span>@RatingValue stars selected</span>
+    </div>
     <h5>Disabled</h5>
-    <BitRating DefaultRating="5" IsEnabled="false" />
+    <BitRating IsEnabled="false" @bind-Value="RatingDefaultValue" />
     <h5>ReadOnly</h5>
-    <BitRating DefaultRating="5" IsReadonly="true" />
+    <BitRating IsReadonly="true" @bind-Value="RatingDefaultValue" />
     <h5>Custom icons</h5>
-    <BitRating Icon="HeartFill" UnselectedIcon="Heart" />
+    <BitRating Icon="HeartFill" UnselectedIcon="Heart" @bind-Value="RatingDefaultValue" />
```

Comment:
There is not any sample in which the Value is changed from outside, like the BitCheckBox sample.
for example, there could be a button and its click handler changes the value property of a BitRating.